### DISCUSSION TITLE
Double Title in docs

### DISCUSSION THREAD

**Discussion by msasl** [facebook/docusaurus#7038] [Q&A]
Hi,

updating from **2.0.0-beta.15** to **2.0.0-beta.18**  all went smooth except for documents in /docs.
It now displays the page file name as title **+** the # title.

Eg:
![image](https://user-images.githubusercontent.com/74366232/160356277-78eed11f-0cdb-407a-ad98-1c6e95973889.png)

Expected behavior:

I have the file "test.mdx" as follows:

`# Test

this is a test`

and usually my page was rendered like follows:

`
# Test

this is a test
`

so how to get rid of the page file name as title and just use the # one?

**Comment by Josh-Cena** [comment]
Hi, I can't seem to reproduce this. Do you have a repro? Our website has pages that use `title` front matter, and those that use the h1 element, but on none of the pages are the file name displayed.

**Reply by Josh-Cena** [reply]
Hi, I've found the cause. It's probably because of CRLF line break characters. I'll send a fix.
